.travis.yml

```diff
@@ -49,4 +49,4 @@ matrix:
       after_failure:
         - echo "Linting failed"
     - env:
-      - TEST="all-settings"
+      - TEST="integration"
```

Comment:
Can you add multiple `TEST` variables here? One for each test? That way all the workflows are tested separately.

Reply:
I'm actually a bit confused the original worked at all, since there is not tag "all-settings", that's just the name of one of the tests.

Reply:
I've added the new tests I wrote as a separate env so they are run independently